.terraform.lock.hcl

```diff
@@ -0,0 +1,22 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
This file was auto generated upon running "terraform init". It is generally recommended to keep this file in the repository, so that all members of the team will leverage the same specific provider/module versions.

It acts similarly to the `package-lock.json` file when working with NodeJS projects.